src/server.ts

```diff
@@ -4,12 +4,16 @@ import morgan from 'morgan';
 const app = express();
 const port = 3000;
 const logger = morgan('short');
+const cors = require('cors');
```

Comment:
This can be `import cors from "cors"` after installing the types, and would be good to move it near the top with other imports